src/main/java/seedu/timetable/TimetableDto.java

```diff
@@ -0,0 +1,113 @@
+package seedu.timetable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class TimetableDto {
+    private ArrayList<TimetableItem> monday;
+    private ArrayList<TimetableItem> tuesday;
+    private ArrayList<TimetableItem> wednesday;
+    private ArrayList<TimetableItem> thursday;
+    private ArrayList<TimetableItem> friday;
+    private ArrayList<String> modulesTaken;
+    private int totalMcTaken;
+
+    /**
+     * Create Timetable DTO.
+     * 
+     * @param modulesTaken list of modules taken
+     * @param totalMcTaken total mcs taken
+     * @param schedule map of day to schedule items
+     */
+    public TimetableDto(ArrayList<String> modulesTaken, int totalMcTaken,
+            HashMap<String, ArrayList<TimetableItem>> schedule) {
+        this.modulesTaken = modulesTaken;
+        this.totalMcTaken = totalMcTaken;
+        for (String day : schedule.keySet()) {
+            ArrayList<TimetableItem> daySchedule = schedule.get(day);
+            switch (day) {
+            case "Monday":
+                this.monday = daySchedule;
+                break;
+            case "tuesday":
+                this.tuesday = daySchedule;
+                break;
+            case "wednesday":
+                this.wednesday = daySchedule;
+                break;
+            case "thursday":
+                this.thursday = daySchedule;
+                break;
+            case "friday":
+                this.friday = daySchedule;
```

Comment:
Might want to keep this case standardised. Also include saturday and maybe sunday as there are modules with lessons on those days as well